<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncodeWindow</class>
 <widget class="QDialog" name="EncodeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EncodeWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: black</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Mono</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1D1B26;
border-radius: 4px;
color: white;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Введите сообщение для шифровки:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>411</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1D1B26;
  text-decoration: none;
  color: white;
  display: inline-block;
  position: relative;
  padding: 15px 30px;
  border: 1px solid;
  border-image: linear-gradient(180deg, #ff3000, #ed0200, #ff096c, #d50082);
  border-image-slice: 1;
  margin: 10px 20px;
  font-family: 'Montserrat', sans-serif;
  text-transform: uppercase;
  overflow: hidden;
  letter-spacing: 2px;
  transition: .8s cubic-bezier(.165, .84, .44, 1);
</string>
   </property>
   <property name="text">
    <string>Зашифровать</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_fromfile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Mono</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:4px;
background: transparent;</string>
   </property>
   <property name="text">
    <string>Выбрать ключи из файла</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_introducedkey">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Mono</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:4px;
colour: white;
background: transparent;</string>
   </property>
   <property name="text">
    <string>Указать ключи вручную</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Mono</family>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1D1B26;
text-decoration: none;
color: white;
border-radius:4px;
</string>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1D1B26;
text-decoration: none;
color: white;
border-radius:4px;
</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>369</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1D1B26;
text-decoration: none;
color: white;
border-radius:4px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1D1B26;
border-radius: 4px;
color: white;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Импортировать из файла&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PT Mono</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1D1B26;
border-radius: 4px;
color: white;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Ввести ключ вручную&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
